compiler+runtime/src/cpp/jank/runtime/obj/chunked_cons.cpp

```diff
@@ -186,6 +186,11 @@ namespace jank::runtime
     return runtime::to_string(seq());
   }
 
+  native_persistent_string obj::chunked_cons::to_code_string() const
+  {
+    return to_string();
```

Comment:
I don't think this is correct. For sequences, we'll need to use `to_code_string` on all of the elements. Should be able to see this with:

```clojure
user=> (cons "foo bar" '("spam"))
("foo bar" "spam")
```